SpamPkg/Core/Runtime/SpamResponderReport.c

```diff
@@ -23,10 +23,10 @@
 #include <Library/PeCoffLib.h>
 #include <Library/PeCoffLibNegative.h>
 #include <Library/SafeIntLib.h>
-#include <Library/TpmMeasurementLib.h>

```

Comment:
I see you removed this lib - I was going to say you can remove it form the INF too, but I don't see it in the inf. Do you know how this was even working in the first place?